sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -414,7 +414,6 @@ object FunctionRegistry {
     expression[MapFromArrays]("map_from_arrays"),
     expression[MapKeys]("map_keys"),
     expression[MapValues]("map_values"),
-    expression[MapEntries]("map_entries"),
```

Comment:
To remove `map_entries`, we need to R together. I'll include my PR to you, @cloud-fan .
cc @felixcheung

Reply:
Also, we need to remove `map_entires` from `functions.scala`, too.

Reply:
Also, cc @HyukjinKwon . We need to remove `map_entries` in Python.